src/frequenz/dispatch/_bg_service.py

```diff
@@ -244,24 +246,40 @@ async def _run(self) -> None:
                     heappop(self._scheduled_events).dispatch
                 )
             elif selected_from(selected, stream):
-                _logger.debug("Received dispatch event: %s", selected.message)
-                dispatch = Dispatch(selected.message.dispatch)
-                match selected.message.event:
-                    case Event.CREATED:
-                        self._dispatches[dispatch.id] = dispatch
-                        await self._update_dispatch_schedule_and_notify(dispatch, None)
-                        await self._lifecycle_events_tx.send(Created(dispatch=dispatch))
-                    case Event.UPDATED:
-                        await self._update_dispatch_schedule_and_notify(
-                            dispatch, self._dispatches[dispatch.id]
-                        )
-                        self._dispatches[dispatch.id] = dispatch
-                        await self._lifecycle_events_tx.send(Updated(dispatch=dispatch))
-                    case Event.DELETED:
-                        self._dispatches.pop(dispatch.id)
-                        await self._update_dispatch_schedule_and_notify(None, dispatch)
-
-                        await self._lifecycle_events_tx.send(Deleted(dispatch=dispatch))
+                match selected.message:
+                    case StreamStartedEvent():
+                        _logger.info("Dispatch stream restarted, refreshing dispatches")
```

Comment:
"(re)started"?

Reply:
Realistically we never will get the start notification because we create the `new_receiver()` only after it was sent

Reply:
Unless the channel's `resend_latest` is `True` and there are no messages in between. But point taken.

Reply:
I think I'll actually will set it to resend_latest=true, it makes for a more consistent experience and also simplifies some parts on the dispatch side

Reply:
Really? What can it simplify? In the general case it will only give you the last dispatch update received in the channel,, you'll always need to go get a fresh list of dispatches anyway, receiving the latest one doesn't really bring much, does it?

Reply:
Found a different approach now